packages/frontend/core/src/modules/editor-setting/impls/user-db.ts

```diff
@@ -25,9 +25,8 @@ export class CurrentUserDBEditorSettingProvider
         key,
         value,
       });
-    } else {
-      this.fallback.set(key, value);
     }
+    this.fallback.set(key, value);
```

Comment:
Electron need to read the editor setting on startup. There is no easy way in the electron side to know where to read the editor setting value (global state json or from sqlite db?)
This is a workaround to make sure even if user db is enabled, the settings will also written to global state json, which should be available when electron is starting.

Reply:
Since we can't enable and disable spell-check dynamically, how about move it out of the editor setting?